doxygen/lang/245_parse_directives.dox.tmpl

```diff
@@ -262,18 +262,47 @@
     Use old pre-0.8.12 precedence where the \ref logical_and_operator "&&" had the same precedence as \ref logical_or_operator "||" and all three bitwise operators (
     \ref bitwise_and_operator "&", \ref bitwise_or_operator "|", \ref bitwise_xor_operator "^") shared the same precedence.
 
+    An example of impact of @ref broken-logic-precedence "%broken-logic-precedence" directive:
```

Comment:
this does not render correctly - you can use a table, but not in this format - you need to remove the -----s for example